docs/products/mysql/howto/connect-using-mysqlx-with-python.rst

```diff
@@ -0,0 +1,61 @@
+Connect to MySQL using MySQLx with Python
+=========================================
+
+Enabling MySQLx protocol support allows customers to use their MySQL instance as a document store. This example shows how to connect to your Aiven for MySQL instance using MySQLx protocol.
+
+.. note::
+
+    MySQL initially provided support for X-DevAPI (MySQLx) in v5.7.12 as an optional extension that can be installed by the user. On the MySQL v8.0+, the X-DevAPI is supported by default.
+
+Variables
+'''''''''
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Variable
+     - Description
+   * - ``SERVICE_URI``
+     - Connection information about your Aiven for MySQLx service.
+   * - ``MYSQLX_PASSWORD``
+     - Password for ``avnadmin`` user.
```

Comment:
I'd remove "avnadmin", since the name of the user can be different. Can be "Password for the user"

Reply:
I think better to put user  before the password, then it is more obvious to which user the password belongs

Reply:
ok, done